libs/isograph-react/src/core/IsographEnvironment.ts

```diff
@@ -125,14 +126,17 @@ export type DataId = string;
 
 export const ROOT_ID: DataId & '__ROOT' = '__ROOT';
 
-export type IsographStore = {
+export type DataLayer = {
   [index: TypeName]: {
     [index: DataId]: StoreRecord | null;
   } | null;
+};
+
+export interface IsographStore extends DataLayer {
```

Comment:
`BaseNodeData` and `NodeData` (above)? It's confusing that `environment.store` is not a `Store`, and `node.data` is a `Store`

We could rename this when we export it, if `BaseNodeData` is too technical. But IMO it's fine.